usaspending_api/download/migrations/0004_download_views.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 2.2.23 on 2021-06-10 13:36
+from pathlib import Path
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("accounts", "0001_initial"),
+        ("awards", "0001_initial"),
+        ("download", "0003_auto_20180306_1726"),
+        ("financial_activities", "0001_initial")
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="AccountBalancesView",
+            fields=[],
+            options={
+                "db_table": "vw_account_balances",
+                "managed": False
+            }
+        ),
+        migrations.RunSQL(
+            sql=[f"{Path('usaspending_api/download/sql/account_balances.sql').read_text()}"],
+            reverse_sql=["DROP VIEW IF EXISTS vw_account_balances;"],
```

Comment:
Oooo. I really like this. I didn't know you could just point django migrations at a sql file. Clean way to version control stuff.

Reply:
Oh, you can't do that, but are just reading a file. Clever.